WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -214,14 +214,17 @@ public static async Task<RpcStatus> GetRpcStatusAsync(this IRPCClient rpc, Cance
 			return newEstimations;
 		}
 
-		public static async Task<(bool accept, string rejectReason)> TestMempoolAcceptAsync(this IRPCClient rpc, IEnumerable<Coin> coins)
+		public static async Task<(bool accept, string rejectReason)> TestMempoolAcceptAsync(this IRPCClient rpc, IEnumerable<Coin> coins, int fakeOutputsCount = 1)
 		{
 			// Check if mempool would accept a fake transaction created with the registered inputs.
 			// This will catch ascendant/descendant count and size limits for example.
 			var fakeTransaction = rpc.Network.CreateTransaction();
 			fakeTransaction.Inputs.AddRange(coins.Select(coin => new TxIn(coin.Outpoint)));
 			Money fakeOutputValue = NBitcoinHelpers.TakeAReasonableFee(coins.Sum(coin => coin.TxOut.Value));
-			fakeTransaction.Outputs.Add(fakeOutputValue, new Key());
+			for (int i = 0; i < fakeOutputsCount; i++)
+			{
+				fakeTransaction.Outputs.Add(fakeOutputValue, new Key());
```

Comment:
Wont work. This will create a tx something like `1->0.99, 0.99, 0.99, 0.99, 0.99, 0.99, 0.99, 0.99`.

Reply:
Yes, I am getting bad-txns-in-belowout.

Reply:
I am working on it.

Reply:
fixed

Reply:
An idea: decide on a small value for output, but for all intent and purposes it should be above the dust thresholds. Create `x` outputs of this above dust value kind. x = requested output number - 1. Then create the last output with `TakeAReasonableFee(coins.sum - x outputs sum)` and you should be good.

Reply:
The current solution is not good? I just div the total with number of outputs.

Reply:
Yes, even cleaner. And you take the fee fair and square.
![image](https://user-images.githubusercontent.com/9156103/82119992-237b1400-9783-11ea-97f8-7902fe19a738.png)

Reply:
I didn't expect communists to dance under this comment but now I see I was wrong.

Reply:
They never do.